src/main/java/com/eva/model/applicant/ApplicationStatus.java

```diff
@@ -0,0 +1,54 @@
+package com.eva.model.applicant;
+
+import static com.eva.commons.util.AppUtil.checkArgument;
+import static java.util.Objects.requireNonNull;
+
+import java.util.Arrays;
+
+
+/**
+ * Represents the application status of an applicant - accepted, rejected or processing.
+ */
+public class ApplicationStatus {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Application Status should only contain the words: processing, accepted, rejected. It should not be blank";
+    public static final String[] POSSIBLE_STATUSES = {"processing", "accepted", "rejected"};
```

Comment:
Should we include pre-interview? I think processing means after the interview but no update yet right?

Reply:
Agree with Royce. Maybe we can use an enum or separate class for this as well.